client/rhel/spacewalk-client-tools/spacewalk-client-tools.spec

```diff
@@ -25,7 +25,7 @@
 %global __python /usr/bin/python3
 %endif
 
-%if !(0%{?rhel} >= 8 || 0%{?sle_version} >= 150000 )
+%if !(0%{?rhel} >= 8 || 0%{?suse_version} >= 1500 )
```

Comment:
Wouldn't this still build on Tumbleweed? What's the value of suse_version there?